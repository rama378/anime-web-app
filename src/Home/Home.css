.Home {
    min-height: 500px;
    padding: 0 0 32px 0;
    background-color: var(--background-light);
}
