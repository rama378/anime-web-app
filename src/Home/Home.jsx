import Search from './components/Search/Search';
import './Home.css';

const Home = () => {
  return (
    <div className='Home'>
      <Search />
    </div>
  );
}

export default Home;
