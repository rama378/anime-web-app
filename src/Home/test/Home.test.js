import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Home from '../Home';

const renderer = new ShallowRenderer();

describe('<Home />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<Home />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
