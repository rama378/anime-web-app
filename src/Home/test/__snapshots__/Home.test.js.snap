// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render and match the snapshot 1`] = `
<div
  className="Home"
>
  <Search />
</div>
`;
